src/storage/client/StorageClient.cpp

```diff
@@ -544,15 +545,15 @@ StorageClient::lookUpVertexIndex(GraphSpaceID space,
                            });
 }
 
-folly::SemiFuture<StorageRpcResponse<storage::cpp2::LookUpEdgeIndexResp>>
+folly::SemiFuture<StorageRpcResponse<storage::cpp2::LookUpIndexResp>>
 StorageClient::lookUpEdgeIndex(GraphSpaceID space,
                                IndexID indexId,
                                std::string filter,
                                std::vector<std::string> returnCols,
                                folly::EventBase *evb) {
```

Comment:
Here , add A new parameter isEdge as better? so we can unify the similar client api. WTYT?

Reply:
Because I have support `is_edge` in `LookUpIndexRequest` so we can merge the two interface into one. I think it's a good idea :D